lib/common/cpu.h

```diff
@@ -50,6 +51,41 @@ MEM_STATIC ZSTD_cpuid_t ZSTD_cpuid(void) {
             f7c = (U32)reg[2];
         }
     }
+#else
+    #error NOCOMMIT: Test to see if we have CI coverage for Windows + clang
```

Comment:
I will delete this line before merging.

This is here to test to see if we have CI coverage for Windows + clang. If we don't, I will attempt to add CI coverage, so we can ensure that this works.

I have reasonably high confidence that it will work, given that it is copied from folly. But it is possible that it doesn't even compile because of a typo.

Reply:
We do have Visual Studios ClangCL coverage: https://github.com/facebook/zstd/actions/runs/8254616105/job/22579195705?pr=3957